### PR TITLE
feat: public roadmap document

### PR DESCRIPTION
**Pull request by piotrmsc**
Updating roadmap document

### CODE REVIEW

**Review by aeneasr — APPROVED** [review]

File: docs/docs/early-access.mdx [hunk 1]
```diff
@@ -3,35 +3,76 @@ id: early-access
...
 ## Roadmap
 
-We have several things on the roadmap for this year:
+Users are often asking when a particular feature will be implemented or when it
+can be expected to be available. This document provides our high level plans in
+a more time frame focus manner.
+
+We will also create Github Project Board in
+[Ory Cloud GitHub repository](https://github.com/ory/cloud) with issues and
+timeliness reflecting our plan presented down below. Issues will provide more
```
Comment (vinckr) [inline docs/docs/early-access.mdx:27]:
```suggestion
timelines reflecting our plan presented down below. Issues will provide more
```

File: docs/docs/early-access.mdx [hunk 2]
```diff
@@ -3,35 +3,76 @@ id: early-access
...
-5. Documentation is lacking.
-6. Advanced configuration of Ory Kratos is currently not possible.
+:::
 
 ## Roadmap
 
-We have several things on the roadmap for this year:
+Users are often asking when a particular feature will be implemented or when it
+can be expected to be available. This document provides our high level plans in
+a more time frame focus manner.
```
Comment (vinckr) [inline docs/docs/early-access.mdx:23]:
```suggestion
can be expected to be available. This document outlines high-level plans for the Ory Cloud.
```

**Comment by aeneasr** [comment]
Whoops, my bad @vinckr !

**Comment by aeneasr** [comment]
I thought it was just copy/paste from the internal docs 😓 but looks like there are still some changes you wanted to have - if you open a new pr we can review/merge

**Comment by piotrmsc** [comment]
it was but Vincent spotted 2 things that were not cached before :)

**Pull request merged**